roaster-tm4j/roaster-tm4j-dto/src/main/resources/xsd/hu/icellmobilsoft/roaster/tm4j/dto/super.xsd

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:jaxb="http://java.sun.com/xml/ns/jaxb"
+    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+    xmlns="http://super.dto.icon.icellmobilsoft.hu/super"
```

Comment:
es sem tukrozi a konyvtarat, tegyuk bele a tm4j nevet is a namespace-be, lathato lessz hova tartozik